gravitee-apim-gateway/docker/Dockerfile

```diff
@@ -15,13 +15,14 @@
 #
 
 # First stage to share environment variable
-FROM graviteeio/java:17-alpine-3.18 AS base
+FROM graviteeio/java:21 AS base
 ENV GRAVITEEIO_HOME=/opt/graviteeio-gateway
 
 RUN apk update  \
     && apk add --no-cache libc6-compat  \
     && apkArch="$(apk --print-arch)" \
-    && ln -s /lib/libc.musl-${apkArch}.so.1 /lib/ld-linux-${apkArch/_/-}.so.2
+    && if [ $(apk --print-arch) = "aarch64" ]; then ln -s /lib/libc.musl-aarch64.so.1 /lib/ld-linux-aarch64.so.2; fi
```

Comment:
Can use the variable assigned previously no?
`&& if [ $(apkArch) = "aarch64" ]; then ln -s /lib/libc.musl-aarch64.so.1 /lib/ld-linux-aarch64.so.2; fi` 

Out of curiosity, does it mean that there is no need for an additional link for other architectures?

Reply:
should we also have a look to this PR in the same time ?
https://github.com/gravitee-io/gravitee-api-management/pull/9857

Reply:
> Can use the variable assigned previously no? `&& if [ $(apkArch) = "aarch64" ]; then ln -s /lib/libc.musl-aarch64.so.1 /lib/ld-linux-aarch64.so.2; fi`
> 
> Out of curiosity, does it mean that there is no need for an additional link for other architectures?

I did it on the other DockerFile and forgot about this one. I fixed it.